app/presenters/details_presenter.rb

```diff
@@ -0,0 +1,41 @@
+require "govspeak"
+
+module Presenters
+  class DetailsPresenter
+    attr_reader :content_item_details, :body
+
+    def initialize(content_item_details)
+      @content_item_details = SymbolizeJSON.symbolize(content_item_details)
+      @body = content_item_details[:body]
+      @body = [@body] if @body.is_a?(Hash)
+    end
+
+    def details
+      @details ||= presented_details
+    end
+
+  private
+
+    def presented_details
+      return content_item_details unless can_render_govspeak?
+      govspeak = { content_type: "text/html", content: rendered_govspeak }
+      content_item_details.merge(body: body + [govspeak])
+    end
+
+    def can_render_govspeak?
+      return false unless body.respond_to?(:any?)
```

Comment:
maybe we can drop line 32 if we do that on this line, kill two birds with one stone type thing?